lib/db/client-metrics-db.js

```diff
@@ -55,6 +57,20 @@ class ClientMetricsDb {
             .orderBy('created_at', 'asc')
             .map(mapRow);
     }
+
+    getInterval() {
```

Comment:
similar comment to https://github.com/Unleash/unleash/pull/450/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.lock#r295445476